src/auxiliary.jl

```diff
@@ -121,55 +138,82 @@ Return the coefficient of `a::TaylorN`, specified by
 function get_coeff(a::TaylorN, v::Array{Int,1})
```

Comment:
Rename to `getcoeff` and/or remove?

Reply:
Since this is basically the same as `getindex`?

Reply:
I have renamed it as proposed; sometimes it is more practical than using the vector notation.